tests/integrations/basic-fail/Cargo.stdout

```diff
@@ -17,7 +17,7 @@ tests/actual_tests/rustc_ice.rs ... FAILED
 tests/actual_tests/bad_pattern.rs FAILED:
 command: "rustc" "--error-format=json" "--extern" "basic_fail=$DIR/$DIR/../../../target/$TMP/$TRIPLE/debug/libbasic_fail.rlib" "--extern" "basic_fail=$DIR/$DIR/../../../target/$TMP/$TRIPLE/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/$DIR/../../../target/$TMP/$TRIPLE/debug" "-L" "$DIR/$DIR/../../../target/$TMP/$TRIPLE/debug" "--out-dir" "$TMP "tests/actual_tests/bad_pattern.rs" "--edition" "2021"
 
-error: substring `miesmätsched types` not found in stderr output
+error: `miesmätsched types` not found in diagnostics for line 5
```

Comment:
were you trying to refer to the line where the error occurs? because that's line 4. This information is currently not available where you're reporting the error.

Reply:
I was referring to the line where the error is expected to occur. That is line 5, right? The error was "not found in line 5". Saying it was "not found in line 4" would be wrong since as you say it *was* found in line 4.

Reply:
so... the annotation is in line 5, but it has a `~^`, so it refers to line 4. But the message was not found in line 4.

Reply:
I think I fixed this.